### PR TITLE
Feature/command

### PR DESCRIPTION
**Pull request by djames9**
Initial working commands, I am finishing pass, nick, user, quit, ping. now lionel is working on private message. and we were thinking after that we should start channel.

### CODE REVIEW

File: test/command/testCommandClass.cpp
Comment (liocle) [inline test/command/testCommandClass.cpp:1]: Do we need to merge that file?

File: src/common/magicNumber.h
Comment (liocle) [inline src/common/magicNumber.h:1]: The three last define could be in the section : "// Numeric reply names and numbers"

**Review by liocle — APPROVED** [review]
Not sure if the comments I have made should be handled prior the merge... you decide!

**Review by mxafi — APPROVED** [review]
Reviewed face to face, needs some improvement and removal of unused or dead code.

File: src/common/magicNumber.h
Comment (mxafi) [reply src/common/magicNumber.h:1]: I agree, that can be handled in a clean-up pull request seperately. For now, just to get the ball rolling, we can approve and merge this. This PR enables us to work on commands, which was not possible before now.

**Pull request merged**